website/source/docs/audit/file.html.md

```diff
@@ -70,6 +70,12 @@ Following are the configuration options available for the backend.
             A boolean, if set, enables the hashing of token accessor. Defaults
             to `true`. This option is useful only when `log_raw` is `false`.
       </li>
+	  <li>
+        <span class="param">mode</span>
+        <span class="param-flags">optional</span>
+            An unsigned integer, if provided, represents the file mode and
```

Comment:
This isn't an integer, though -- it's a string representation of an integer in base 8. I'd say something like "A string containing an octal number representing the bit pattern for the file mode, similar to chmod."